sdk/core/src/services/WalletStorageService.ts

```diff
@@ -51,18 +54,23 @@ type ConstructorOptions = {
 export default class WalletStorageService {
   private _storageRegion: string
   private _keysService: KeysService
+  private _didAuthService: DidAuthService
   private _platformEncryptionTools: IPlatformEncryptionTools
+  private _vaultApiService: AffinidiVaultApiService
   private _bloomVaultApiService
 
   constructor(
     keysService: KeysService,
+    didAuthService: DidAuthService,
     platformEncryptionTools: IPlatformEncryptionTools,
     options: ConstructorOptions,
   ) {
     this._keysService = keysService
+    this._didAuthService = didAuthService
     this._platformEncryptionTools = platformEncryptionTools
-    this._bloomVaultApiService = new BloomVaultApiService(options)
+    this._bloomVaultApiService = new BloomVaultApiService({ ...options, vaultUrl: options.bloomVaultUrl })
```

Comment:
Or better to do it explicitly now, when there are several vault urls: `new BloomVaultApiService({ accessApiKey: options.accessApiKey, vaultUrl: options.bloomVaultUrl })`